docs/source/install.rst

```diff
@@ -23,11 +23,23 @@ Source
 
 The following instructions assume you'll be using ucx-py on a CUDA enabled system and is in a `Conda environment <https://docs.conda.io/projects/conda/en/latest/>`_.
 
-Note: UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.
+.. note::
+    UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, ``librdmacm``,
```

Comment:
Oops missed this. Apparently we are referencing `librdmacm` twice. I'll update in PR ( https://github.com/rapidsai/ucx-py/pull/351 ).